src/atoms/atomgraph.jl

```diff
@@ -129,7 +85,6 @@ function AtomGraph(
     input_file_path::String,
     id::String = "",
```

Comment:
I was just playing with something earlier and realized it would be nice if this defaulted to the filename (without extension) when building from file, rather than a blank string. Could you add that?

Reply:
Resolved in 7257df3.